source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm

```diff
@@ -509,37 +509,29 @@ - (void)loadImage:(std::string const &)urlStr
         url = [NSURL URLWithString:nSUrlStr relativeToURL:_hostConfig.baseURL];
     }
 
-    NSObject<ACOIResourceResolver> *imageResourceResolver = [_hostConfig getResourceResolverForScheme:[url scheme]];
-    ImageLoadBlock imageloadblock = nil;
-    if (!imageResourceResolver || ![imageResourceResolver respondsToSelector:@selector(resolveImageResource:)]) {
-        imageloadblock = ^(NSURL *url) {
-            // download image
-            UIImage *img = nil;
-            if ([url.scheme isEqualToString:@"data"]) {
-                NSString *absoluteUri = url.absoluteString;
-                std::string dataUri = AdaptiveCards::AdaptiveBase64Util::ExtractDataFromUri(std::string([absoluteUri UTF8String]));
-                std::vector<char> decodedDataUri = AdaptiveCards::AdaptiveBase64Util::Decode(dataUri);
-                NSData *decodedBase64 = [NSData dataWithBytes:decodedDataUri.data() length:decodedDataUri.size()];
-                img = [UIImage imageWithData:decodedBase64];
-            } else {
-                img = [UIImage imageWithData:[NSData dataWithContentsOfURL:url]];
-            }
-            return img;
-        };
-    }
+    ImageLoadBlock imageloadblock = ^(NSURL *url) {
```

Comment:
You skipped the resolver. will it impact the user if they https://docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/ios/extensibility#adcresolver

Reply:
this is a dead code. I removed it. user of resolver won't call this method.